CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/ValueInputPane.java

```diff
@@ -139,6 +139,14 @@ public ValueInputPane(final PluginParameter<?> parameter, int defaultWidth, Inte
                     recentValuesCombo.setDisable(true);
                 }
 
+                if (recentValues != null) {
+                    if (recentValues.size() > 1) {
+                        parameter.setStringValue(recentValues.get(1));
+                    } else {
+                        parameter.setStringValue(recentValues.get(0));
```

Comment:
if-else can be converted to tertiary (similar to suggestion for `ColorInputPane`)